.travis.yml

```diff
@@ -6,8 +6,6 @@ notifications:
   webhooks: http://basho-engbot.herokuapp.com/travis?key=8f07584549e458d4c83728f3397ecbd4368e60a8
   email: eng@basho.com
 otp_release:
-  - R16B02
-  - R16B01
-  - R15B03
-  - R15B02
-  - R15B01
+  - 18.1
+  - 17.5
+  - R16B03
```

Comment:
Is this true? AFAIK we are still on R16B02 as our supported version. Of course it's a Basho special...